sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dotenv
```

Comment:
Same question, why we need this file?

Reply:
It's used to store and configure API endpoint and key. This is also widely used in this repo. But if you prefer not to use it, i can remove it for this PR.